clients/da-vinci-client/src/main/java/com/linkedin/davinci/stats/IngestionStats.java

```diff
@@ -328,51 +328,51 @@ public void recordConsumedRecordEndToEndProcessingLatency(double value) {
   }
 
   public double getRecordsConsumed() {
-    return recordsConsumedSensor.getRate();
+    return recordsConsumedSensor.measure(METRIC_CONFIG, System.currentTimeMillis());
   }
 
   public void recordRecordsConsumed() {
     recordsConsumedSensor.record();
   }
 
   public double getBytesConsumed() {
-    return bytesConsumedSensor.getRate();
+    return bytesConsumedSensor.measure(METRIC_CONFIG, System.currentTimeMillis());
   }
 
   public void recordBytesConsumed(double value) {
-    bytesConsumedSensor.record(value);
+    bytesConsumedSensor.record((long) value);
   }
 
   public double getLeaderRecordsConsumed() {
-    return leaderRecordsConsumedSensor.getRate();
+    return leaderRecordsConsumedSensor.measure(METRIC_CONFIG, System.currentTimeMillis());
```

Comment:
Should we add a `getRate()` function in LARG that does this boilerplate, rather than change every call-site?